checker/manual/fenum-checker.tex

```diff
@@ -120,9 +118,24 @@ \section{Running the Fenum Checker\label{fenum-running}}
 
 \begin{alltt}
   javac -processor org.checkerframework.checker.fenum.FenumChecker
-        \textit{-Aquals=myproject.qual.MyFenum} MyFile.java ...
+        \textit{-Aquals=qual.MyFenum} MyFile.java ...
 \end{alltt}
 
+This option specifies a comma-no-space-separated list of the fully-qualified
```

Comment:
Please rephrase, because the option may syntactically be a comma-no-space-separated list, but that isn't what it specifies.